pandas/tests/tools/test_to_numeric.py

```diff
@@ -77,6 +77,30 @@ def test_series(last_val):
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    "list_data,kwargs",
+    [
+        (["-3.14", 7], {}),
+        (
+            ["-3.14", 7, pd.to_datetime(0), pd.NaT, ["30", -10]],
+            {"errors": "coerce"}
+        ),
+        (
+            ["-3.14", 7, pd.to_datetime(0), pd.NaT, ["30", -10]],
+            {"errors": "ignore"}
+        ),
+    ]
+)
+def test_list_series(list_data, kwargs):
+    lis = list_data
+    ser = Series(list_data)
+
+    result = to_numeric(lis, **kwargs)
+    expected = to_numeric(ser, **kwargs).values
+
+    tm.assert_numpy_array_equal(result, expected)
```

Comment:
why are you comparing vs a numpy array?

these should use assert_series_equal at a minimum

Reply:
Because one input was a `list` and the other input was a `pd.Series`. So the first ouput is a `np.array` and the output for `series.values` is a `np.array` when `series` is instance of `pd.Series`. It could have been done in other way tough;
```python
...
result = to_numeric(lis, **kwargs)
expected = to_numeric(ser, **kwargs)

tm.assert_series_equal(pd.Series(result), expected)
```